common-arcana.lic

```diff
@@ -365,41 +365,81 @@ module DRCA
     end
   end
 
-  def update_astral_data(data)
+  def update_astral_data(data, settings)
     if data['moon']
       data = set_moon_data(data)
     elsif data['stats']
-      data = set_planet_data(data)
+      data = set_planet_data(data, settings)
     end
     data
   end
 
-  def find_visible_planets(planets)
-    return if DRC.bput('get my telescope', 'You get', 'What were you', 'You are already') == 'What were you'
+  def find_visible_planets(planets, settings)
+
+    telescope_available = false
+    if settings.have_telescope
+      telescope_available = true
+    end
+
+    # TODO: respect YAML telescope_storage container
+    if telescope_available
+      case DRC.bput('get my telescope', 'You get', 'What were you', 'You are already', 'need a free hand')
```

Comment:
❓ How is this different than https://github.com/rpherbig/dr-scripts/blob/master/common-moonmage.lic#L27 ?

Reply:
Yeah, the DRMM is what should be used, simply for the variety of ways that people access their telescopes, and the more obvious reason is code duplication.

Reply:
Sure, happy to update this script to use DRMM. I had no idea that existed.